frontend/README.md

```diff
@@ -1,3 +1,23 @@
 # Frontend
 
-* We are planning to use react for now. Open for suggestions.
\ No newline at end of file
+- create-react-app to bootstrap the react application.
+    create-react-app --template typescript
+
```

Comment:
Why does it have info regarding bootstrapping an react application ? I assume it is done only once.

Reply:
Yep it's needed only once, but just added in the readme if we need it.